operators/pkg/controller/common/defaults/pvc.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package defaults
+
+import v1 "k8s.io/api/core/v1"
```

Comment:
Nit: `v1` alias is useless here. We used to use `corev1`.

Reply:
I know, but `make fmt` adds it back in again if I remove it :(

We're very inconsistent about `v1` vs `corev1` in the rest of the code base. I think we're going to need a cleanup-PR and a CI check to ensure consistency in the long term here.